tests/conftest.py

```diff
@@ -33,7 +33,40 @@
 NucypherClickConfig.log_to_file = False
 
 # Crash on server error by default
-WSGIController._crash_on_error_default = False
+WebEmitter._crash_on_error_default = False
+
+
+##########################################
+
+@pytest.fixture(autouse=True, scope='session')
+def __very_pretty_and_insecure_scrypt_do_not_use():
```

Comment:
I need a "tuxxy" emoji here.

Reply:
Haha wow. Why not use "insecure" parameters for real scrypt though?

Reply:
It probably wouldn't matter, but at least would preserve exactly the same formatting as Scrypt originally has (just in case)

Reply:
You mean, the content-format of the derived material bytes?

Reply:
> You mean, the content-format of the derived material bytes?

Yes. Obviously, it will be insecure to use in practice, but ok in tests, while running the same Scrypt code (so we may catch some errors which we'd otherwise miss)

Reply:
I cannot imagine what error can be caused in this way.  I opted to mock SCrypt entirely, so that we dont test it.  If you think we need to, we can make a single SCrypt integration test.

Reply:
I think it's fine as it is, that's ok

Reply:
I actually like this Idea, since, as you always remind me that "everything that is not tested will eventually fail".